README.md

```diff
@@ -15,11 +15,25 @@ GoCert requires 3 files to operate:
 * A private key
 * A TLS certificate with that private key
 * A YAML config file with the required parameters
-as an example:
+
+You can generate the cert and the associated key by running:
+`openssl req -newkey rsa:2048 -nodes -keyout key.pem -x509 -days 1 -out cert.pem -subj "/CN=example.com"`
+GoCert does not support insecure http connections.
+
+### Config File
+The config file requires the following parameters:
+* keypath: this is the path to the private key you've generated.
+* certpath: this is the path to a certificate for enabling HTTPS connections.
+* dbpath: the path to a sqlite database file. If the file does not exist GoCert will attempt to create it.
+* port: the port in integer form to serve all of GoCert's API and frontend
+* pebblenotificationsenabled: a boolean that once enabled, will allow GoCert to send pebble notices. Read more about it (here)[https://github.com/canonical/pebble?tab=readme-ov-file#notices].
```

Comment:
```suggestion
* pebble_notify: Allow GoCert to send pebble notices on certificate events (create, update, delete). Read more about it (here)[https://github.com/canonical/pebble?tab=readme-ov-file#notices]. (bool)
```

Reply:
Took all of the suggestions and converted this into a table. [Here's](https://github.com/canonical/gocert/blob/TLSENG-239-pebble-notices/README.md) the preview.